account_vat_period_end_statement/__manifest__.py

```diff
@@ -0,0 +1,35 @@
+#  Copyright 2011-2012 Domsense s.r.l. (<http://www.domsense.com>)
+#  Copyright 2012-17 Agile Business Group (<http://www.agilebg.com>)
+#  Copyright 2012-15 LinkIt Spa (<http://http://www.linkgroup.it>)
+#  Copyright 2015 Associazione Odoo Italia (<http://www.odoo-italia.org>)
+#  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "ITA - Liquidazione IVA",
+    "version": "12.0.1.0.0",
+    'category': 'Localization/Italy',
+    'license': 'AGPL-3',
```

Comment:
Se hai tempo potresti aggiungere una breve riga di "summary"

Reply:
Grazie @primes2h per la review.
Non ricordo dove sono le linee guida per le traduzioni e (mettendomi anche un po' nei panni di uno nuovo) non le trovo:

1. Dal README del repository ho _Vedi la nostra [documentazione per i traduttori](https://github.com/OCA/l10n-italy/wiki/Traduzioni)_.
2. Dal link precedente mi dice di andare invece in [https://www.odoo-italia.org/contribuire/traduzioni](https://www.odoo-italia.org/contribuire/traduzioni)
3. Dal link precedente (pagina 'Partecipare') mi dice _Buona conoscenza delle [linee guida](https://github.com/OCA/l10n-italy/wiki/Traduzioni/#linee-guida)_ che rimanda al link del punto 2..

In sintesi, dove trovo le linee guida? Grazie!

Reply:
Hai ragione, non era per nulla chiaro.
Ho ripristinato una parte della pagina wiki semplificando anche la pagina del gruppo, prova ora e fammi sapere.

Buona parte di quelle informazioni andranno comunque nella documentazione ufficiale, ma finchè non definiamo bene il tema e la struttura di base (*) meglio non complicare le cose.

(*) ci sarebbe da revisionare questa, chi vuole si faccia pure avanti :yum: 
https://github.com/odoo-italia/documentazione/pull/6